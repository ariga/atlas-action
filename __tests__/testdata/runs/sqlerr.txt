{
  "Env": {
    "Driver": "sqlite3",
    "URL": {
      "Scheme": "sqlite",
      "Opaque": "",
      "User": null,
      "Host": "test",
      "Path": "",
      "RawPath": "",
      "ForceQuery": false,
      "RawQuery": "mode=memory&cache=shared&_fk=1",
      "Fragment": "",
      "RawFragment": "",
      "DSN": "file:test?mode=memory&cache=shared&_fk=1",
      "Schema": "main"
    },
    "Dir": "__tests__/testdata/bad-sql"
  },
  "Steps": [
    {
      "Name": "Detect New Migration Files",
      "Text": "Found 1 new migration files (from 1 total)"
    },
    {
      "Name": "Detect New Migration Files",
      "Text": "Failed loading changes on dev database",
      "Error": "executing statement: near \"bad\": syntax error"
    }
  ],
  "Schema": {},
  "Files": [
    {
      "Name": "1_ver.sql",
      "Error": "executing statement: near \"bad\": syntax error"
    }
  ]
}