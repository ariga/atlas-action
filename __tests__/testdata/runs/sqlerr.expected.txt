<h1>Atlas Lint Report</h1>

<table><tr><th>Status</th><th>Step</th><th>Result</th><th>Diagnostics</th></tr><tr><td><div align="center"><img src="https://release.ariga.io/images/assets/success.svg" /></div></td><td>Detect New Migration Files</td><td>Found 1 new migration files (from 1 total)</td><td></tr><tr><td><div align="center"><img src="https://release.ariga.io/images/assets/error.svg" /></div></td><td>Detect New Migration Files</td><td>Failed loading changes on dev database</td><td>executing statement: near "bad": syntax error</td></tr><tr><td><div align="center"><img src="https://release.ariga.io/images/assets/special-warning-icon.svg" /></div></td><th colspan="3">Connect your project to <a href="https://auth.ariga.cloud/login">Ariga Cloud</a> to get more safety checks</th></tr></table>
